pkgs/misc/tmux-plugins/tmux-fingers/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  mkTmuxPlugin,
+  substituteAll,
+  fetchFromGitHub,
+  crystal,
+  shards,
+}:
+let
+  fingers = crystal.buildCrystalPackage rec {
+    format = "shards";
+    version = "2.2.2";
+    pname = "fingers";
+    src = fetchFromGitHub {
+      owner = "Morantron";
+      repo = "tmux-fingers";
+      rev = "${version}";
+      sha256 = "sha256-m9QON7diHVEDnnv/alXCJOG+BnfrAKygScrubZZ605I=";
+    };
+
+    shardsFile = ./shards.nix;
+    crystalBinaries.tmux-fingers.src = "src/fingers.cr";
+
+    postInstall = ''
+      shopt -s dotglob extglob
+      rm -rv !("tmux-fingers.tmux"|"bin")
+      shopt -u dotglob extglob
+    '';
+
+    doCheck = false;
+    doInstallCheck = false;
```

Comment:
Maybe add a comment why tests won't work.

Reply:
Thanks, I'm getting the following error with doCheck:

```
@nix { "action": "setPhase", "phase": "checkPhase" }
Unhandled exception: Missing ENV key: "TMUX" (KeyError)
  from /nix/store/1cjf5grfqsjbjvqadsdf8l1nhinj474s-crystal-1.11.2-lib/crystal/env.cr:57:7 in 'fetch'
  from /nix/store/1cjf5grfqsjbjvqadsdf8l1nhinj474s-crystal-1.11.2-lib/crystal/env.cr:22:5 in '[]'
  from src/fingers/dirs.cr:5:15 in '~Fingers::Dirs::TMUX_PID:init'
  from src/fingers/dirs.cr:6:3 in '__crystal_main'
  from /nix/store/1cjf5grfqsjbjvqadsdf8l1nhinj474s-crystal-1.11.2-lib/crystal/crystal/main.cr:129:5 in 'main_user_code'
  from /nix/store/1cjf5grfqsjbjvqadsdf8l1nhinj474s-crystal-1.11.2-lib/crystal/crystal/main.cr:115:7 in 'main'
  from /nix/store/1cjf5grfqsjbjvqadsdf8l1nhinj474s-crystal-1.11.2-lib/crystal/crystal/main.cr:141:3 in 'main'
  from /nix/store/wlffq5p6mxxgfap10sav3ij936jzqm59-glibc-2.39-52/lib/libc.so.6 in '??'
  from /nix/store/wlffq5p6mxxgfap10sav3ij936jzqm59-glibc-2.39-52/lib/libc.so.6 in '__libc_start_main'
  from /build/source/.crystal/crystal-run-spec.tmp in '_start'
  from ???
```